### PR TITLE
VForm

### PR DESCRIPTION
**Pull request by Shinigami92**
https://vuetifyjs.com/en/components/forms

### CODE REVIEW

File: src/components/VForm.tsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { ofType } from 'vue-tsx-support';
+import { VForm } from 'vuetify/lib';
+
+export default ofType<Props>().convert(VForm as any);
+
+type Props = {
+  ref?: string;
+  'v-model'?: any;
```
Comment (Shinigami92) [inline src/components/VForm.tsx]: I'm not sure if this can be `boolean`

Comment (chanlito) [reply src/components/VForm.tsx]: Hey @Shinigami92 I'm not sure if we should be putting `v-model` on Props here nor `ref`.

Comment (Shinigami92) [reply src/components/VForm.tsx]: I thought this could be helpfull for autocompletion

Comment (chanlito) [reply src/components/VForm.tsx]: I think `vue-tsx-support` expose `ref` but not `v-model` since with a vue thing not tsx specific.

Comment (Shinigami92) [reply src/components/VForm.tsx]: Should I remove this line or can you cherry-pick some things?

File: src/components/VForm.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { ofType } from 'vue-tsx-support';
+import { VForm } from 'vuetify/lib';
+
+export default ofType<Props>().convert(VForm as any);
```
Comment (chanlito) [inline src/components/VForm.tsx]: Maybe `ofType<Props, CommonEvents>` here?

File: src/components/VForm.tsx (hunk 1 above)
Comment (chanlito) [reply src/components/VForm.tsx]: yes remove it.

**Pull request merged**